src/RageTimer.cpp

```diff
@@ -88,19 +91,12 @@ float RageTimer::GetDeltaTime()
 	return diff;
 }
 
-/*
- * Get a timer representing half of the time ago as this one.  This is
- * useful for averaging time.  For example,
- *
- * RageTimer tm;
- * ... do stuff ...
- * RageTimer AverageTime = tm.Half();
- * printf( "Something happened approximately %f seconds ago.\n", tm.Ago() );
- */
+// Revert to the original StepMania 3.95 code.
```

Comment:
This comment doesn't tell me anything besides that I now need to dig up where to find the 3.95 code and cross reference the function. Just keep the original comment mang, which also exists in the 3.95 code base.